dipper/curie_map.yaml

```diff
@@ -1,3 +1,7 @@
+# Skolemize Blank Nodes
+# overwrite iri fragment with '' for unresovable bnodes
+'_':    'https://monarchinitiative.org/.well-known/genid/'
```

Comment:
+1  dipper produces RDF but should strongly limit its notion of blank nodes to be no more than OWL allows. 
<https://en.wikipedia.org/wiki/Blank_node#Anonymous_classes_in_OWL> 

    anonymous classes such as unions or intersections of classes, or classes called restrictions, defined by a constraint on a property.